starter-code/Movies/routes/celebrities.js

```diff
@@ -0,0 +1,97 @@
+const express = require("express");
+
+// require the Celebrity model here
+const Celebrity = require("../models/celebrity");
+const router = express.Router();
+
+router.get("/celebrities", (req, res, next) => {
+  Celebrity.find({}, (err, celebrities) => {
+    if (err) {
+      return next(err);
+    }
+    console.log(celebrities);
+    res.render("celebrities/index", {
+      celebrities: celebrities
```

Comment:
In ES6 you can just create 
{ celebrities } and it will be the same as { celebrities: celebrities}.
It's the same but cleaner syntax, if you want to use it :)
: